.env

```
@@ -1,26 +1,2 @@
-# Eureka Configuration
-EUREKA_URL=http://localhost:9999/eureka/
-EUREKA_HOSTNAME=localhost
-
-# JWT Configuration (for Identity Service)
-JWT_SECRET=mySecretKey12345678901234567890123456789012345678901234567890
-JWT_EXPIRATION=86400000
-
-# Inter-service secret
-INTER_SERVICE_SECRET=inter-service-secret-key-123456789
+GANACHE_MNEMONIC="gadget evoke oil anchor able cake lift drop fork useful once owner"
+ACCOUNT_PRIVATE_KEY=0xaba9dede35ae3042e09949a82605a33f77b1602a6cc463ee7b829d663accf570
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)